webview_panels/src/lib/altimate/main.d.ts

```
@@ -0,0 +1,597 @@
+
+export declare interface ViewsCodeModalArgs {
+    table: string;
+    column: string;
+    viewsType: ViewsTypes;
+    viewsCode: Record<string, [string, string][]>;
+    nodeType: string;
+}
+
+declare type ViewsTypes = keyof typeof VIEWS_TYPE_COLOR;
+
+export { }
```

Biome

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)